integration/storage/base.py

```diff
@@ -284,6 +292,16 @@ def _create_tempfile(self, prefix='', content=b''):
             self.addCleanup(os.remove, path)
             return path
 
+        @classmethod
+        def _random_container_name(cls):
+            suffix = random_string(cls.container_name_max_length)
+            name = cls.container_name_prefix + suffix
```

Comment:
Yeah, I think including a special prefix on which we can filter on in the "clean up phase" is a good idea.